core/object/script_language_extension.h

```diff
@@ -492,6 +492,25 @@ class ScriptLanguageExtension : public ScriptLanguage {
 			}
 		}
 	}
+	virtual void debug_get_stack_level_locals(int p_level, Vector<String> *p_locals, Array *p_values, int p_max_subitems = -1, int p_max_depth = -1) override {
+		Dictionary ret;
+		GDVIRTUAL_REQUIRED_CALL(_debug_get_stack_level_locals, p_level, p_max_subitems, p_max_depth, ret);
+		if (ret.size() == 0) {
```

Comment:
```suggestion
		if (ret.is_empty()) {
```

Reply:
Why is this?

Reply:
Because `is_empty` is preferred over `size() == 0` while doing the same thing, we use it most of the time 🙂

Reply:
Ohhhh omg I saw/thought like I used `.is_empty()` and you wanna change it to `.size() == 0` hehe.